controllers/gcpcluster.go

```diff
@@ -117,15 +124,64 @@ func (r *GCPClusterReconciler) reconcileNormal(ctx context.Context, gcpCluster *
 }
 
 func (r *GCPClusterReconciler) reconcileDelete(ctx context.Context, gcpCluster *capg.GCPCluster) (ctrl.Result, error) {
-	err := r.firewallClient.DeleteBastionFirewallRule(ctx, gcpCluster)
+	err := r.deleteBastionFirewallRule(ctx, gcpCluster)
 	if err != nil {
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
-	err = r.client.RemoveFinalizer(ctx, gcpCluster, FinalizerFW)
+	err = r.client.RemoveFinalizer(ctx, gcpCluster, FinalizerFirewall)
 	if err != nil {
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
 	return ctrl.Result{}, nil
 }
+
+func (r *GCPClusterReconciler) createBastionFirewallRule(ctx context.Context, logger logr.Logger, gcpCluster *capg.GCPCluster) error {
+	ruleName := fmt.Sprintf("allow-%s-bastion-ssh", gcpCluster.Name)
```

Comment:
as this `sprintf` for ruleName repeat on multiple places I would rather see it somewhere as function

Reply:
Also done, thanks!